internal/view/pod.go

```diff
@@ -295,6 +296,24 @@ func shellIn(a *App, fqn, co string) {
 	}
 }
 
+func debugIn(a *App, fqn, co string) {
+	os, err := getPodOS(a.factory, fqn)
+	if err != nil {
+		log.Warn().Err(err).Msgf("os detect failed")
+	}
+
+	image := defaultDebugImage
+	if a.Config.K9s.DebugImage != "" {
```

Comment:
I think this is a bad place where we should check DebugImage and assign the default value. Is it possible to move it to config.go somewhere?  Thanks!

Reply:
Yeah, I'll give it another try. I originally tried adding the default to https://github.com/derailed/k9s/blob/master/internal/config/k9s.go#L42 but the default value would never come through.

I _think_ https://github.com/derailed/k9s/blob/master/internal/config/config.go#L224-L226 might end up clobbering the default value placed as part of `NewK9s` but I wasn't sure so I went this route.

Reply:
Got it.

I think what had happened is I added the setting without a default in the first attempt, so I wound up with `debugImage: ""` in my config. Let me know if the updated location makes sense to you.